tests/qinfo/test_entropies.py

```diff
@@ -24,9 +24,7 @@ def expected_entropy_ising_xx(param):
     """
     Return the analytical entropy for the IsingXX.
     """
-    eig_1 = (1 + np.sqrt(1 - 4 * np.cos(param / 2) ** 2 * np.sin(param / 2) ** 2)) / 2
-    eig_2 = (1 - np.sqrt(1 - 4 * np.cos(param / 2) ** 2 * np.sin(param / 2) ** 2)) / 2
-    eigs = [eig_1, eig_2]
+    eigs = [np.cos(param / 2) ** 2, np.sin(param / 2) ** 2]
```

Comment:
I love this! I was confused about the change but the math works out perfectly

Reply:
I don't even remember how we arrived at that complicated expression before, since the density matrix is already diagonal. 😄